make/jdk/src/classes/build/tools/depend/Depend.java

```diff
@@ -878,24 +880,30 @@ public Void visitModifiers(ModifiersTree node, Void p) {
             return super.visitModifiers(node, p);
         }
 
+        @Override
+        public Void visitPrimitiveType(PrimitiveTypeTree node, Void p) {
+            update(node.getPrimitiveTypeKind().name());
```

Comment:
shouldn't `DependTest` include a test to stress this new code?

Reply:
Test updated:
https://github.com/openjdk/jdk/pull/12801/commits/de84439d64f6882d64f657fee0f6c55b2f717a9c

Thanks!